scripts/build/testdata/build_all_except_host.txt

```diff
@@ -2,13 +2,13 @@
 cd "{root}"
 
 # Generating ameba-amebad-all-clusters
-bash -c '$AMEBA_PATH/project/realtek_amebaD_va0_example/GCC-RELEASE/build.sh {root} ninja {out}/ameba-amebad-all-clusters all-clusters-app'
+bash -c '$AMEBA_PATH/project/realtek_amebaD_va0_example/GCC-RELEASE/build.sh  {root} ninja {out}/ameba-amebad-all-clusters all-clusters-app'
```

Comment:
extra space should not be here - we try to make the `--dry-run` output be reasonably human-readable

Reply:
Understood, this has been fixed.